db/migrate/20240617105409_remove_oval_permissions.rb

```diff
@@ -0,0 +1,16 @@
+class RemoveOvalPermissions < ActiveRecord::Migration[6.1]
+  def up
+    Permission.where(resource_type: 'ForemanOpenscap::OvalPolicy').destroy_all
+    Permission.where(resource_type: 'ForemanOpenscap::OvalContent').destroy_all
+    Permission.where(resource_type: 'ForemanOpenscap::Cve').destroy_all
```

Comment:
Not sure why anyone would do that, but would this work if anyone has a filter set up that uses any of these permissions? Iirc in that case we may need to drop things from the filterings table first.